.tekton/tasks/rebase-pr.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rebase-pr
+spec:
+  description: |
+    rebase the pull request for which the test is running on staging.
+  workspaces:
+   - name: infra-deployment-git
+   - name: shared-file-path
+  steps:
+    - name: rebase-pull-request
+      image: quay.io/redhat-appstudio/github-app-token@sha256:b4f2af12e9beea68055995ccdbdb86cfe1be97688c618117e5da2243dc1da18e
+      workingDir: $(workspaces.infra-deployment-git.path)
+      script: |
+        #!/usr/bin/env bash
+        set -eufx
+
+        git config --global --add safe.directory $(pwd)
+        git fetch
+        git config --global user.email "prietyc123@gmail.com"
```

Comment:
@Roming22 to use the git operation for rebase we nedd to provide the user mail and name where `staging` github app is the user here. So do you know what is the user and name for the github app. If we don't provide these then we get the error:
```
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'root@rhtap-staging-e2e-n6gp9-rebase-pr-pre-test-pod.(none)')
```

Reply:
You can use anything (e.g. `rhtap-ci-bot@redhat.com` and `RHTAP CI Robot`). I do not believe those values are validated against anything.